### ISSUE TITLE
train on default setting on qa1 but accuracy stay about 20%

### ISSUE THREAD

**Issue by tomsonsgs** [therne/dmn-tensorflow#12]
![default](https://user-images.githubusercontent.com/30223858/31889205-8a60eb38-b831-11e7-9868-a6c99ad36841.JPG)

**Comment by michael-wzhu** [comment]
yes, i also observe this problem. Do you have some thoughts about how to solve it?

**Comment by michael-wzhu** [comment]
I have switched the initialization method from "He et al " to Xavier, the accuracy immediately increased a lot. 

And i found that the configs have some differences from the original paper, such as learning rate.

**Comment by JaneShenYY** [comment]
@michael-wzhu I have got the same problem. Would you please specify how did you modify the code? and what accuracy for task1 did you obtain after the modification please?
